src/invidious/database/users.cr

```diff
@@ -50,6 +50,7 @@ module Invidious::Database::Users
   end
 
   def mark_watched(user : User, vid : String)
+    mark_unwatched(user, vid)
```

Comment:
Can you change the SQL query below to handle everything in a single DB request?

Reply:
I'm not sure how to do that, not the Postgres expert

Reply:
```sql
UPDATE users
SET watched = array_append(array_remove(watched, $1), $1)
WHERE email = $2
```